__tests__/app.test.js

```diff
@@ -317,3 +317,27 @@ describe("/api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("/api/users", () => {
+  test("GET 200: should return array of all users with required properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const users = body.users;
```

Comment:
We need to add a length check for the users coming back, before we run our forEach